sdk/healthinsights/azure-healthinsights-radiologyinsights/CHANGELOG.md

```diff
@@ -1,14 +1,21 @@
 # Release History
 
-## 1.0.0b2 (Unreleased)
+## 1.0.0 (Unreleased)
```

Comment:
Just a reminder, before releasing the package make sure to update this entry to include your release date